cmd/crc/cmd/daemon.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
would it be possible to be explicit about what we compile it for?

Reply:
i could change the file name to `daemon_nonwin.go`